app/src/androidTest/README.md

```
@@ -0,0 +1,119 @@
+| 5 | GPRS speed throttle | Speed → GSM | Speed → full |
+| 6 | Disconnect then slow reconnect | Disconnect → 5s → connect at GSM → 5s | Speed → full |
+| 7 | Flap then long disconnect (20s) | Disconnect → connect → disconnect → 20s | Network connect |
+| 8 | Latency flapping | 3x: 5s delay 2s → 0 delay 2s | Delay 0 |
+| 9 | Speed degradation cycle | HSDPA → 3s → EDGE → 3s → GSM → 5s | Speed → full |
+| 10 | Disconnect + degraded recovery | Disconnect → 10s → connect with 3s delay + EDGE → 5s | Delay 0 + speed full |
+
+## Test output
+
+The test prints a summary at the end:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)